data/json/items/armor.json

```diff
@@ -1147,7 +1147,39 @@
     "warmth": 30,
     "material_thickness": 2,
     "environmental_protection": 1,
-    "flags": [ "VARSIZE" ]
+    "flags": [ "VARSIZE" ],
+    "use_action": {
+      "menu_text": "Zip up",
+      "type": "transform",
+      "target": "bondage_mask_zipped",
+      "msg": "You zip up the eyes and mouth of the bondage mask."
+    }
+  },
+  {
+    "id": "bondage_mask_zipped",
+    "type": "ARMOR",
+    "name": "bondage mask (zipped)",
+    "description": "A tight mask made of black leather. The eyes and mouth have been closed using zippers.",
+    "weight": 210,
+    "volume": 4,
+    "price": 21000,
```

Comment:
I don't know its real price, but more than 200 bucks for a mask looks like too much.

Reply:
I've done a quick google search, and they seem to be about $30-50, so I've update the price to $40.